### PR TITLE
Added "Troubleshooting" section to README

### PR DESCRIPTION
**Pull request by victor-soloviev**
Added information for resolving the issue with OpenSSL build that the
team faced several times.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ Cargo.toml:
 onvif-rs = "0.1"
 ```
 
+## Build
```
Comment (zeenix) [inline README.md]: I think this is more a troubleshooting section than build. Wouldn't you agree?

**Review by zeenix — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -19,6 +19,18 @@ Cargo.toml:
...
+
+If you have an issue with OpenSSL build under Ubuntu, perform the following actions:
+
+```
+sudo apt install openssl1.0 libssl1.0.0 libssl1.0-dev
+cargo clean
+OPENSSL_LIB_DIR="/usr/lib/x86_64-linux-gnu"
+OPENSSL_INCLUDE_DIR="/usr/include/openssl"
```
Comment (DmitrySamoylov) [inline README.md]: Do you need `export` here?

Comment (victor-soloviev) [reply README.md]: You are right, in a command line it should be done with export. I'll fix text.

File: README.md (hunk 1 above)
Comment (victor-soloviev) [reply README.md]: Makes sense

**Review by zeenix — APPROVED** [review]
:+1:

**Review by DmitrySamoylov — APPROVED** [review]

**Pull request merged**